flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/controller/FlinkSessionJobController.java

```diff
@@ -266,4 +276,14 @@ private Optional<String> validateSessionJob(FlinkSessionJob sessionJob, Context
         }
         return validationError;
     }
+
+    private void addMetricsIfAbsent(FlinkSessionJob sessionJob) {
```

Comment:
Same comment as I added to the deployment controller

Reply:
fixed